developer_guide/akeneo_in_a_nutshell/index.rst

```diff
@@ -0,0 +1,79 @@
+Akeneo PIM in a nutshell
+========================
+
+What is a PIM?
+**************
+
+A Product Information Management (PIM) solution is aimed to centralize all the marketing data, to enrich, translate and prepare it for exports to multiple channels.
+It is a productivity tool helping the contributors to serve the product information in different languages and for different purposes.
+
+Akeneo is the first open source PIM made for non-technical users.
+Easy to use and very flexible, the solution can adapt to your product organization and processes, not the other way around.
+Available in two editions, Community, and Enterprise, you can choose the one that fits your needs.
+
+What is a product?
+------------------
+
+Akeneo provides a powerful and very flexible way to structure your products using an Entity Attribute Value (i.e. Product - Attribute - ProductValue) model.
+
+For instance, let's say we want to create a "Car" product in Akeneo.
```

Comment:
> what you describe here ("Car") is the family

Yes! I introduce the "Family" concept: sounds logical to me, I will reword this part to be more clear on what is the "Car".

Disagree on "identifier" part here, because in all systems you have an unique identifier, so I guess the beginner is easy with this concept, what do you think ?